src/java/org/apache/nutch/tools/AbstractCommonCrawlFormat.java

```diff
@@ -23,6 +23,7 @@
 import java.net.URLEncoder;
 import java.net.UnknownHostException;
```

Comment:
@lewismc 
As you had noticed it was previously formatted with tabs. I reformatted to 2 spaces.
I used IntelliJ Idea to reformat, I hope that followed all the style rules in `eclipse-codeformat.xml`. 
Please review and also let me know if you find any issues.